docs/_posts/2016-01-11-admin-manual.md

```diff
@@ -115,14 +115,18 @@ and populate it with the following information:
       username="tgaclims"
       password="tgaclims"/>
       <Parameter name="miso.propertiesFile" value="file:${catalina.home}/conf/Catalina/localhost/miso.properties" override="false"/>
-      <Parameter name="miso.name" value="Test"/>
+      <Parameter name="miso.instanceName" value="Your MISO instance name"/>
+      <!-- uncomment if using Runscanner -->
+      <!--
+      <Parameter name="runscanner.configFile" value="/etc/runscanner.json" override="false"/>
```

Comment:
It looks like this is only used by RunScanner itself. Is there a reason MISO needs to know about it?

If so, it's not in our stage or production config...